### ISSUE TITLE
App Icon

### ISSUE THREAD

**Issue by willbrazil** [wguedes01/subtexting-app#1]
As you can certainly see. There's no real icon for the app. Feel free to design one if you have time. I'll be more than happy to use it! :)

**Comment by PierBover** [comment]
What resolution do you need?

**Comment by willbrazil** [comment]
I think DPI's ~160, ~240, and ~320 (http://developer.android.com/design/style/iconography.html) would be enough for most devices that will be using the app. I just have no idea what the icon should look like :) If you are creative and would like to help, I'd love to add your work to the project.

**Comment by willbrazil** [comment]
Here's the current icon: hahah

![](https://lh3.googleusercontent.com/uXUl3FO-qr6sNJlR5bhqEGs6j-PrNLTXBsU9IXZP8lgSsb0gEikY5OKLAetUPiVTsVu3=w300-rw)

**Comment by jshgyn** [comment]
Here's a quick attempt from me. If you're interested in using it I'll upload all the sizes you need.

![512x512](https://cloud.githubusercontent.com/assets/8613892/7334568/41172b48-eb8f-11e4-99f6-b595c2dc43ee.png)

Josh

**Comment by PierBover** [comment]
Here's my quick take on it. It's a mix between the Sublime Text and the Hangouts icons.

![image](https://cloud.githubusercontent.com/assets/226483/7335097/92ecf1e0-eb6d-11e4-98b0-8689246fd111.png)

**Comment by axynos** [comment]
Make it material, add long shadow and you're golden.

**Comment by PierBover** [comment]
Hey @axynos what do you mean by material? Does material design specify constraints for icons?

**Comment by leogg** [comment]
![subtexting_icon](https://cloud.githubusercontent.com/assets/2543067/7336184/ba298522-ebac-11e4-9936-646ff408d6f6.png)

**Comment by axynos** [comment]
@PierBover Generally material design is focused on bold, graphic design. That means that there is very few places where gradients are allowed to be used if you follow their material philosophy(I've linked it below). You could also add a long shadow to your logo's S and you're golden.

Here's the material design guidelines.
http://www.google.com/design/spec/material-design/introduction.html#introduction-principles

**Comment by sdvim** [comment]
![subtexting-app-logo](https://cloud.githubusercontent.com/assets/2823088/7337046/82c9b62a-ebce-11e4-9b77-d4812735365e.png)

**Comment by axynos** [comment]
I absoloutely love it. Goes with the theme of sublime very well.

**Comment by willbrazil** [comment]
@sdvim, I think it looks fantastic. Like @axynos mentioned, I really like the theme. I want to bring this up to hear everyone's thoughts: Do you think the shape of the "bubble" looks too much like Google Hangout's? [see below] Again, I really liked it!

![](https://chromebookparadise.files.wordpress.com/2013/05/hangouts-logo.png?w=584&h=584)

**Comment by sdvim** [comment]
Glad they're liked so far! I made some extra variations in case the community wanted to see more options:

![subtexting-app-logos](https://cloud.githubusercontent.com/assets/2823088/7359463/6bfe34e0-ecf3-11e4-8996-2c9d700aade3.png)

**Comment by willbrazil** [comment]
Wow. They're fantastic! I think the 4th one would fit the best with the app. What do you think?

**Comment by sdvim** [comment]
I agree! It feels enough like the original Sublime logo with a subtle hint of the Hangouts logo, but moreover a standard chat application.

**Comment by lefoy** [comment]
+1 for the 4th one (bottom right) :)

**Comment by purduekenny** [comment]
I think it's a toss up between the first and fourth - either one with the chat bubble provides the necessary affordance and visual clue it's a chat.

**Comment by corysimmons** [comment]
1st looks best but 4th isn't a ripoff of the Hangouts logo.

Not sure what the project is. If it has anything to do with Hangouts, use 1. If it doesn't, use 4.

**Comment by willbrazil** [comment]
@corysimmons Thanks for the feedback! The project is about a Sublime Text plugin that lets you send and receive text messages.

**Comment by corysimmons** [comment]
Then yeah don't rip off Hangouts. I'd go with 4. =)

**Comment by naikrovek** [comment]
N° 4 is excellent.
